packages/utils/src/arrayFilters.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Type safe helper function to filter out undefined items of an array
+ */
+export const notUndefined = <T>(item?: T): item is T => typeof item !== 'undefined';
+
+/**
+ * Helper function to filter only distinct elements of an array
+ */
+export const distinct = <T>(txid: T, index: number, self: T[]) => self.indexOf(txid) === index;
```

Comment:
In the `@trezor/utils` package we are following the structure of one file per utility function, so in this case it should be 2 different files one for `notUndefined` and other for `distinct`. With the file having same name of the function.

When we make it in different files it might be nice to change the name from `distinct` to `arrryDistinct` so it makes more clear that is intended use is for array as we do with other functions in this package like `arrayPartition`.

In the case of `notUndefined` I do not think that it is strictly related to array so maybe name would be nicer to have `isNotUndefined` to have a name that gives more information about what the function is doing.

The reason for that is to have a better developer experience so when a dev is looking for an utility function does not have to dig into multiple files but just look at the `src/` flat directory.

Reply:
All done. I just have difficulties to force myself to create file with a single line of code :)

Reply:
It's absolutely valid to have files with one line of code, it's easier to find function you need by going through file names that trying to guess in which file (with some name summary name) it is hidden.